sabnzbd/filesystem.py

```diff
@@ -792,23 +792,25 @@ def find_dir(p):
     # windows diskfree
     try:
         # Careful here, because win32api test hasn't been done yet!
-        import win32api
-    except:
-        pass
+        import win32api as w32
+    except ImportError:
+        w32 = None
+
 
     def diskspace_base(_dir):
         """ Return amount of free and used diskspace in GBytes """
         _dir = find_dir(_dir)
         try:
-            available, disk_size, total_free = win32api.GetDiskFreeSpaceEx(_dir)
+            available, disk_size, total_free = w32.GetDiskFreeSpaceEx(_dir)
             return disk_size / GIGI, available / GIGI
-        except:
+        except (w32.error, ModuleNotFoundError, AttributeError):
             return 0.0, 0.0
 
 
 else:
     try:
-        os.statvfs
+        # os.statvfs                # commented this because it does not appear to do anything.
```

Comment:
What this does is just to see if `statvfs` is available, if not the `try` will catch it and we skip this function.

Reply:
Thanks!  I was wondering about that! I will drop the # and add a comment to that effect.

Reply:
I modified this to a straight function call to ensure Python actually calls it and doesn't ignore it with lazy-loading or whatever happens under the hood.  It's a simple call to the cwd now to make sure it's actually doing something.  It can always be reverted later if it causes a slowdown or other issues.